### ISSUE TITLE
Support Finding Elements matching Suffix from schema using includes [SWS-348]

### ISSUE THREAD

**Issue by gregturn** [spring-projects/spring-ws#499]
**[Stephen Duncan Jr](https://jira.spring.io/secure/ViewProfile.jspa?name=jrduncans)** opened **[SWS-348](https://jira.spring.io/browse/SWS-348?redirect=false)** and commented

When using org.springframework.ws.wsdl.wsdl11.DefaultWsdl11Definition, support discovering *Request *Response element from the included schemas when pointing to a schema like:

\<?xml version="1.0" encoding="UTF-8"?>

<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
targetNamespace="http://www.example.com/foo"
elementFormDefault="qualified">

    <xs:include schemaLocation="bar.xsd" />
    <xs:include schemaLocation="baz.xsd" />

</xs:schema>

---
No further details from [SWS-348](https://jira.spring.io/browse/SWS-348?redirect=false)

**Comment by snicoll** [comment]
@jrduncans assuming you raised this issue on our old Jira instance, There hasn't been any interest from the community that I can see and I am wondering if this issue is still actionable. Thanks!

**Comment by jrduncans** [comment]
Yeah, I have no idea what might be relevant anymore about this 17-year old issue 😄.

**Comment by snicoll** [comment]
Thanks for following-up so quickly. I am going to close this as there isn't an owner for the feature anymore and we'd need some feedback before proceeding anyway. If someone is still interested, we can reopen.
